app/controllers/points_controller.rb

```diff
@@ -10,15 +10,37 @@ def create
     end
   end
 
+  def update
+    @point = Point.find(params[:id])
+    @point.attributes = point_params.except(:image)
+
+    if params[:imageEdited] == "true"
```

Comment:
Two problems:

1. Your API should be consistent with casing. Every other parameter is snake case, so you should stick with snake case. Therefore, this should be `image_edited`.
2. It doesn't make sense to send this parameter to the server. If the client knows the image hasn't been edited, then the client simply shouldn't send the image to the server.

Synopsis: You're making this harder than it has to be. Make your API stupid and your clients smart.